### PR TITLE
linknx: bump to new upstream version

### PR DESCRIPTION
**Pull request by tru7**
Signed-off-by: Othmar Truniger <github@truniger.ch>

Maintainer: me
Compile tested: ar71xx & mpc85xx, WNDR3700 & TL-WDR4900, OpenWRT trunk /LEDE trunk
Run tested: ar71xx & mpc85xx, WNDR3700 & TL-WDR4900, OpenWRT trunk /LEDE trunk

Description:
use newest upstream version to overcome a compile problem with gcc v6
switch source format from gz to xz

### CODE REVIEW

**Comment by tru7** [comment]
Unfortunately I forgot to add the package name in the commit. How can I modifiy or just delete the pull request?

**Comment by mhei** [comment]
Just use `git commit --amend` in your feature branch, then edit the commit message. Then use `git push -f`in this branch to force push the changes to Github. This PR is automatically updated.

**Comment by tru7** [comment]
thank you for your support @mhei, that worked out fine. However my sha256 hash differs from Travis. Also in my 3 different LEDE build environments they differ even using the exact same Makefile. I have Ubuntu 16.04 on 32bit, Debian 8 and 9 on 64bit. I took the hash from Debian which was the same for both Debian. How can I reliably determine the right sha256 hash?

**Comment by diizzyy** [comment]
Please don't update the copyright if you're not affiliated with OpenWrt, add your own or leave it as is.

File: net/linknx/Makefile [hunk 1]
```diff
@@ -8,18 +8,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=linknx
-PKG_VERSION:=0.0.1.33
+PKG_VERSION:=0.0.1.34
```
Comment (diizzyy) [inline net/linknx/Makefile]: This is bogus, it's not released yet.

File: net/linknx/Makefile [hunk 2]
```diff
@@ -8,18 +8,18 @@
...
 
 PKG_MAINTAINER:=Othmar Truniger <github@truniger.ch>
 PKG_LICENSE:=GPL-2.0+
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/linknx/linknx.git
-PKG_SOURCE_VERSION:=f627ed44feaea5a1f67c53f2810c44a14c059cef
+PKG_SOURCE_VERSION:=6166831eff645f21a7fd22ec3f7855f36005b8d9
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)
 PKG_SOURCE:=$(PKG_NAME)-${PKG_VERSION}.tar.gz
```
Comment (diizzyy) [inline net/linknx/Makefile]: Switch to xz and update PKG_MIRROR_HASH

**Comment by tru7** [comment]
@diizzyy all done

**Pull request merged**